src/components/Blog/Blog.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import Heading from "../Common/Heading";
+import "./Blog.css";
+
+function Blog() {
+  return (
```

Comment:
memoization

Reply:
Actually, I am not getting why to use Memoization here for the Blog Page

Reply:
Done